### PR TITLE
Split fstrm 0.3.2 into separate bottle

### PR DESCRIPTION
**Pull request by oerdnj**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fstrm.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Fstrm < Formula
+  desc "Frame Streams implementation in C"
+  homepage "https://github.com/farsightsec/fstrm"
+
+  stable do
```
Comment (DomT4) [inline Formula/fstrm.rb]: Don't need a `stable do` block unless you're doing something special in it, like a `stable`-only resource or dependency.

File: Formula/fstrm.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    url "https://dl.farsightsecurity.com/dist/fstrm/fstrm-0.3.2.tar.gz"
+    sha256 "2d509999ac904e48c038f88820f47859da85ceb86c06552e4052897082423ec5"
+  end
+
+  bottle do
```
Comment (DomT4) [inline Formula/fstrm.rb]: CI does this. You can remove this block.

File: Formula/knot.rb [hunk 3]
```diff
@@ -5,29 +5,19 @@ class Knot < Formula
...
-      url "https://dl.farsightsecurity.com/dist/fstrm/fstrm-0.3.2.tar.gz"
-      sha256 "2d509999ac904e48c038f88820f47859da85ceb86c06552e4052897082423ec5"
-    end
   end
 
   bottle do
-    sha256 "e0a392eb217d4707174b5319bd5c87a1b6ec4a1c8a05f2e8513d1b8827fe0a5f" => :sierra
-    sha256 "25af8be672f352e88edfb9f41d5a2fc2ac74e99e333a451c08080fccfa51a509" => :el_capitan
-    sha256 "fc92515ae8adb4bbcc75629fb525d34935686877352d86ee50797485429d1c9c" => :yosemite
+    rebuild 1
```
Comment (DomT4) [inline Formula/knot.rb]: Likewise here, don't mess with the bottles.

File: Formula/knot.rb [hunk 4]
```diff
@@ -5,29 +5,19 @@ class Knot < Formula
...
+    sha256 "7d5c0733e39bdd7b9b215f98a4f3439764cdcb5abd50c2c8e1ce30b555daa7ee" => :sierra
   end
-
+
```
Comment (DomT4) [inline Formula/knot.rb]: Whitespace be here 👻

**Review by DomT4 — CHANGES_REQUESTED** [review]

**Comment by oerdnj** [comment]
@DomT4 Thanks for the review, all comments should be resolved.

**Comment by oerdnj** [comment]
JFTR The reason for the split is that fstrm can be used in other projects that support dnstap:
- knot-resolver
- unbound
- (?)bind

**Comment by fxcoudert** [comment]
Thanks @oerdnj for your contribution to Homebrew!

**Comment by ilovezfs** [comment]
@fxcoudert I'm not sure what went wrong here, but the knot bottle doesn't have the knot config file (`.bottle/etc/knot.conf`). Hence https://github.com/Homebrew/homebrew-core/pull/13025.

**Pull request closed without merging**